src/scope-manager.js

```diff
@@ -280,12 +280,25 @@ var scopeManager = function(state, predefined, exported, declared) {
       var i, j;
       var currentUsages = _current["(usages)"];
       var currentLabels = _current["(labels)"];
+      var labelNameList = Object.keys(currentLabels);
```

Comment:
`labelNameList` is only used by `preferconst` so I'd put it inside the if-block.
My reasoning for these kind of changes is that now `let` exists, one day this code might be made to es6 and if you were using `let`, you'd probably do that?